docs/index.mdx

```diff
@@ -4,17 +4,37 @@ title: '❓ What is Etherlink?'
 
 import Figure from '@site/src/components/Figure';
 
-Etherlink is an EVM-compatible layer-2 blockchain powered by [Tezos Smart Rollup](https://tezos.com/developers/smart-rollups/) technology.
+Etherlink is an EVM-compatible, non-custodial Layer 2 blockchain powered by [Tezos Smart Rollup](https://tezos.com/developers/smart-rollups/) technology.
+It supports Ethereum's ecosystem tools, including wallets and indexers, facilitating seamless integration and the capability to transfer assets to and from other EVM-compatible chains.
+
+Built upon the secure foundation of Tezos layer 1, Etherlink delivers a fast, fair, and (nearly) free experience.
+This permissionless and censorship-resistant environment empowers developers to build and participate in the next generation of decentralized applications.
 
 ### It's fast
 
-**Etherlink has sub-second block confiration times for a snappy experience**. Tezos' 2-block finality guarantee and the speed of its Smart Rollups ensure that transactions are finalized quickly.
+Etherlink is low latency, with sub-second (< 500ms) confirmation times, delivered by the sequencer.
+The Tezos 2-block finality guarantee and the speed of its Smart Rollups ensure that transactions are finalized quickly.
+
+Chain | L2 finality time | Data posted on L1
+--- | --- | ---
+Etherlink | ~ 500ms | ~ 15 seconds
+Optimism Bedrock | ~ [2 seconds](https://community.optimism.io/docs/developers/build/differences/#blocks) | ~ [2 minutes](https://optimistic.etherscan.io/batches)
+Arbitrum One | ~ [300 ms](https://arbiscan.io/) | ~ [7 minutes](https://arbiscan.io/batches)
+
+### It's fair
+
+Etherlink's governance process promotes transparency and fairness in decision-making.
+Token holders can participate in proposing and voting on changes to the protocol with no administrative keys; users retain complete control, keeping their assets free from custodial control and abuse.
```

Comment:
"Token holders" sounds like Etherlink token holders to me whereas it's really bakers on L1. Feel free to resolve.

Reply:
Changed to "Tezos token holders" unless you think it should be "bakers."